content/en/docs/components/pipelines/installation/standalone-deployment.md

```diff
@@ -40,7 +40,7 @@ Use the [gcloud container clusters create command](https://cloud.google.com/sdk/
 
 CLUSTER_NAME="kubeflow-pipelines-standalone"
 ZONE="us-central1-a"
-MACHINE_TYPE="n1-standard-2" # A machine with 2 CPUs and 7.50GB memory
+MACHINE_TYPE="e2-standard-2" # A machine with 2 CPUs and 8GB memory
```

Comment:
Note that e2 machines cannot have GPUs. Therefore, I think it would help users find relevant information if we also point them to https://cloud.google.com/compute/docs/machine-types for canonical guidance of how to choose the machine type.

Reply:
That is a good point! Updated the instruction.